src/main/java/org/opensearch/ad/transport/ProfileTransportAction.java

```diff
@@ -53,6 +56,8 @@
     private ModelManager modelManager;
     private FeatureManager featureManager;
     private CacheProvider cacheProvider;
+    // the number of models to return. Defaults to 10.
+    private volatile int modelToReturn;
```

Comment:
Suggest to change to `modelsToReturn` here too.

Reply:
changed to numModelsToReturn.